### PR TITLE
v0.27.2-RC3

### PR DESCRIPTION
**Pull request by clanmills**

### CODE REVIEW

**Review by piponazo — DISMISSED** [review]
Great work in this release @clanmills ! 👏

**Comment by clanmills** [comment]
@piponazo I'm still working on this.  I have documentation changes to add.  It will be finished later today.

**Comment by clanmills** [comment]
@piponazo This is v0.27.2-RC3  Please approve and I will merge, build the packages, tag the release and publish.

There has been a lot of very good work done on "dot two".  Well done Everybody.

For v0.27.3, we will only have RC1 (unless there's a bug in RC1 itself).  These "dots" are consuming too much of everybody's time.  The team needs to focus on 0.28 and I should focused on the book.

File: README.md [hunk 1]
```diff
@@ -841,11 +841,13 @@ cmd
...
 <div id="5-6">
 
-### Unix
+### 5.6 Unix
 
-Exiv2 can be built on many Unix and Linux distros.  We actively support the Unix Distributions NetBSD and FreeBSD.
+Exiv2 can be built on many Unix and Linux distros.  With v0.27.2, we are starting to actively support the Unix Distributions NetBSD and FreeBSD.  We hope to add CI support for these platforms in v0.27.3.
 
-I am willing to support Exiv2 on commercial Unix distributions such as Solaris, AIX, HP-UX and OSF/1 provided you provide with an ssh account on your platform.  I will require super-user privileges to install software.
+I have provided notes here based on my experience with these platforms.   Feedback is welcome.
```
Comment (piponazo) [inline README.md]: [minor] We should review this kind of comments for the next release. In my opinion, the main README should contain general information about the project, and not this kind personal statements (Personal pronoun in first singular person)

File: README.md [hunk 2]
```diff
@@ -865,10 +867,29 @@ It's important to ensure that `LD_LIBRARY_PATH` includes `/usr/local/lib` and `/
...
+Clang is pre-installed as ``/usr/bin/{cc|c++}` as well has libz and expat.  FreeBSD uses pkg as the package manager which I used to install cmake and git.
+
+```bash
+$ su root
+Password:
+# pkg install cmake
+# pkg install git
+```
+
+To run the Exiv2 test suite, I installed bash and python.  The test suite requires additional work as the platform `diff` command does not understand the option `--binary` and returns an error.  In consequence, the test harness returns lots of errors.  I hope to address this in v0.27.3.
```
Comment (piponazo) [inline README.md]: Do we have an issue in github for this task?

**Review by piponazo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (clanmills) [reply README.md]: Well, it's how I express myself.  For v0.28, you (or somebody else) can rewrite the documentation if they wish.

File: README.md (hunk 2 above)
Comment (clanmills) [reply README.md]: Yes.  This is discussed in #902 which I will reopen for v0.27.3

**Pull request merged**